docs/src/man/reshaping_new.md

```diff
@@ -0,0 +1,498 @@
+# Reshaping Data
+
+*DataFrames.jl* is a popular julia library for tabular data which makes easier to read and transform data.
+It provides the abstractions of data frames and series, similar to those in *R*, and *Pandas*.
+
+In *DataFrames.jl* data reshaping means the transformaton of the structure of the table or vector (for example 
+data frames or series) to make it suitable for further analysis. Let us assume we have a data frame with 
+multiindices on the rows and columns. In `DataFrames.jl` currently we have two functions: `stack`, and `unstack`
+that we can use for reshaping our data. Their goals are very simple:
+- `stack` allows us to go from wide to long data format;
+- `unstack` allows us to go from long to wide data format. 
+
+!!! note
+
+  Parts of this section has been taken from [Bogumił Kamiński](https://bkamins.github.io/julialang/2021/05/28/pivot.html)
+
+First we will have a look that how to create multiple indices. There is a simplest way to create a multi index 
```

Comment:
DataFrames.jl does not support indices, nor multi indices. You can do `groupby` instead.